ui/src/views/repository-data/components/sync-types-table/columns.tsx

```diff
@@ -10,13 +10,14 @@ import {
   RepositorySyncStatus,
   RepositoryTableRowOptions,
 } from './components'
+import { TimeAgoField } from '../../../../components/Fields/time-ago-field'
 
 export const columns: Array<Record<string, any>> = [
   {
     dataIndex: 'status',
     key: 'syncStateIcon',
     className: "w-12 h-20 p-0",
-    render: ({ syncState }: { syncState : RepSyncStateT }) => (
+    render: ({ syncState }: { syncState: RepSyncStateT }) => (
```

Comment:
Seems to be a typo in `RepSyncStateT` - should be `RepoSyncStateT`?

Reply:
I am agree. Done [3068412](https://github.com/mergestat/fuse/pull/89/commits/3068412459b5ee7151a5b88ad0dc5e880f43f9b0)